docs/guides/deploy.md

```diff
@@ -161,14 +161,14 @@ my_module = MyModule.compile(
 )
 ```
 
-The server can be extracted from the module and saved to a file:
+You can extract the server from the module and save it in a file:
 
 <!--pytest-codeblocks:skip-->
 ```python
 my_module.server.save("server.zip")
 ```
 
-The only noticeable difference with the workflow presented above is the `Client::encrypt`, `Client::decrypt` and `Server::run` methods must be given an extra `function_name` argument specifying the name of the targeted function.
+The only noticeable difference with the deployment of modules and the deployment of circuits is the methods `Client::encrypt`, `Client::decrypt` and `Server::run` must all be given an extra `function_name` argument specifying the name of the targeted function.
```

Comment:
I saw your comment. Do you mean this: 

The only noticeable difference between the deployment of modules and the deployment of circuits is that the methods`Client::encrypt`, `Client::decrypt`, and `Server::run` must contain an extra `function_name` argument specifying the name of the targeted function.

Reply:
Absolutely ! I changed it the way you proposed.